superset-frontend/packages/superset-ui-chart-controls/src/components/labelUtils.tsx

```diff
@@ -55,8 +55,7 @@ const TooltipSection = ({
   text: ReactNode;
 }) => (
   <TooltipSectionWrapper>
-    <TooltipSectionLabel>{label}</TooltipSectionLabel>
-    <span>{text}</span>
+    <TooltipSectionLabel>{label}</TooltipSectionLabel>: <span>{text}</span>
```

Comment:
might be a legit use case for a `&nbsp;`

Reply:
Also not sure the span has any real purpose here. ¯\\\_(ツ)_/¯

Reply:
I had a `{' '}` and I think prettier changed it to what is now there. Generally I prefer the `{' '}` to `&nbsp;`, and if prettier is configured to do what it did, let's roll with it. About the span, I think it's referenced in the CSS so that it's not bolded.